### ISSUE TITLE
Cyber Security : Learning and Careers in Future

### ISSUE THREAD

**Issue by aajinkya1203** [nowmozillaclub/NOW-Mozilla-Blog#13]
> Note this is an article written by [Prathamesh Churi](https://www.linkedin.com/pulse/cyber-security-learning-careers-future-prathamesh-churi/), NOW Mozilla's Mentor.

---

![0](https://user-images.githubusercontent.com/43881544/100242544-ecb85e80-2f5a-11eb-9458-27bef78ae5c5.jpg)

<b>Some random chats from my students with me:*</b><br />

_“Cyber security ?  Huh ! So boring. World is going with Machine learning, Data analytics . . .  So cool stuff on Internet ;), Cyber Security is very old area”_  <br/>

 _“Sir can we do our project in AI  ? , I mean we should go with advance technology, right ?”_ <br/><br/>

_Sir, in which field I should pursue my masters? AI, ML ?_ 

<br/><br/>
Surprisingly, Cyber security is not in their list. Secondly, Cyber security is only “hacking and attacks” for most of them. Students wants to learn attacks and try them with their friends for FUN! The destructive nature of harming someone’s resources makes me feel restless sometimes! Out of 100 students,  hardly 10 students have little-bit interest in cyber security and that too in learning attacks and NOT about control measures on these attacks.<br/><br/>

I always say through my quote -<br/>

<p align="center"><b>Don’t blame Technology, blame people for their mindset !</b></p>

<br/><br/>
This destructive nature itself is the reason, why cyber security is important today as a career option. We must understand that- with the advent of technologies and innovations in AI, ML, Data Analytics, IoT, we require to secure all the assets from malicious things/people/systems. The motivation of learning and pursuing cyber security as a career option is to be smarter in controlling the attacks instead of learning them and use them for bad intentions.<br/><br/>

Nobody is talking about considering Cyber Security as their career option !  I don’t know why? The big companies like Google, Facebook, IBM, Amazon hires many cyber security professionals and pay them a big money in return. Such companies have a big Security team which works silently to secure companies data from malicious contents. Cybersecurity professionals work behind the scenes to protect civilian, business, and government data from hackers.The Cybersecurity subject prepares students for employment in a variety of entry level careers in Cybersecurity with practical hands-on training, necessary to equip students with the skills employers expect.<br/><br/>

In general, students learn :<br/>

1. Configure, troubleshoot and maintain computer systems, virtualised environments and secure network communications.
2. Protect computer networks/systems against unauthorised access, modifications and/or destruction.
3. Analyse and recommend counter measures to mitigate security threats to systems/networks.
4. Develop scripts for automating configuration and management of system/access controls.

<br/><br/>

### Job perspective in Cyber Security Area: 

Cybersecurity professionals work in every size company and industry to protect organisations from data breaches and attacks. And the demand for cybersecurity professionals is growing at a breakneck speed. Job postings for cybersecurity positions have grown three times faster than openings for IT jobs overall. Here are some career pathways you can pursue as a cyber-security professional.<br/><br/>

1. Security Software Developer<br/>

---

Security Software Developers build security software and integrate security into applications software during the design and development process. Depending on the specific position and company, a security software developer might oversee a team of developers in the creation of secure software tools, develop a company-wide software security strategy, participate in the lifecycle development of software systems, support software deployments to customers, and test their work for vulnerabilities.<br/><br/>

2.  Security Architect<br/>

---

If you’re enthusiastic about problem-solving and formulating big-picture strategies, the security architect career path is for you. A security architect is meant to create, build and execute network and computer security for an organisation. Security architects are responsible for developing complex security framework and ensuring that they function effectively. They design security systems to counter malware, hacking and DDoS attacks.<br/><br/>

3. Information Security Analyst<br/>

---

Information Security Analysts are the front-line defence of networks, Information Security Analysts put firewalls and encryption in order to protect breaches, constantly monitor and audit systems for unusual activities.<br/><br/>

4. Ethical Hackers and forensic analysts:
<br/>

---

Ethical hackers normally hold a CEH certificate and are given license by their employers to try and infiltrate the security of their system. The idea is that they use the same techniques as malicious black hat hackers to test existing security protocols; if they are successful, upgrades can then be developed and implemented. <br/>

Forensics analysts focus on cyber-crime, an ever-growing phenomenon. They work with law enforcement agencies in both public and private sector organisations and are asked to undertake a wide variety of tasks, such as Recovering deleted files, Interpreting data linked to crime, Analysing mobile phone records, Pursuing data trails etc.  Computer forensic analysts must keep a well — detailed records of their investigations, and often provide evidence for official purpose .<br/><br/>

5. Chief Information Security Officer (CISO)<br/>

---

The Chief Information Security Officer is normally a mid-executive level position whose job is to manage the affairs operations of a company’s or organisation’s IT security division. CISOs are usually responsible for planning, coordinating and directing all computer, network and data security needs of their employers. CISOs work directly with the management to determine an organisations’ custom cybersecurity demands. The CISOs are usually saddled with the responsibility of assembling an effective staff of security professionals, which means that the position requires an individual with a strong background in IT security architecture and strategy, as well as effective communication and human resource skills.<br/><br/>

6. Penetration Tester<br/>

---

Penetration testing is the proactive authorised employment of testing procedures on the IT system to identify system flaws. A penetration tester usually attempts to (with permission) hack into a computer and network systems to pre-emptively discover operating system vulnerabilities, service and application problems, improper configurations and more, before an intruder cause real damage. Penetration testers must be highly skilled, often using testing tools of their own design, to “break into” the systems under watch. Penetration testers are required to keep accurate records of their activities and discovered vulnerabilities.<br/><br/>

7. IT Security Consultant
<br/>

---

IT security consultants meet with clients to advise them on how to protect their organizations’ cybersecurity objectives best efficiently and cost-effectively. IT Security Consultants are often employed by smaller firms and agencies that cannot afford to handle their security issues in-house but are also employed by big corporation to supplement their security teams and provide an impartial outside perspective to current systems challenges.<br/><br/>

8. Security Systems Administrator<br/>

---

A security systems administrator’s responsibility is a bit similar to many cybersecurity jobs i.e., installing, administering, maintaining and troubleshooting computer, network and data security systems. The main distinction between security systems administrators and other cybersecurity professionals is that the security systems administrator is normally the person in charge of the daily operation of those security systems. The regular tasks include systems monitoring and running regular backups, and setting up, deleting and maintaining individual user accounts. Security systems administrators are usually often involved in developing organisational security procedures.<br/><br/>

In conclusion, As you can see, there are endless paths your cybersecurity career can lead you down. But first, you have to start somewhere. Doing basic certifications, you tube videos and reading some white papers written by cyber security industry people will help you to explore this area more better. <br/>

To end this article, I would like state a quote by Schneier Maxim<br/><br/>

<p align="center">Security is not a product, it’s a process!</p><br/>

— Prof. Patrick

References:

- https://www.cybersecurityeducation.org/career/
- https://medium.com/@coderacademy/careers-in-cyber-security-you-should-consider-2613061a8cb2
- https://www.cyberdegrees.org/resources/transitioning-from-generalit/
